### ISSUE TITLE
Question about the test results of my own testsets with a fine-tuned BERT

### ISSUE THREAD

**Issue by Deep1994** [huggingface/transformers#7038]
# ❓ Questions & Help

## Details

Hi, I have a fine-tuned BERT model and I want to use it to predict on my own testset which have 40k sentence pairs. I use the script like:

`export GLUE_DIR=./glue_data
export TASK_NAME=QQP

CUDA_VISIBLE_DEVICES=3 python ./examples/text-classification/run_glue.py \
    --model_name_or_path ./pretrained_weights/bert_base_uncased \
    --task_name $TASK_NAME \
    --do_predict \
    --data_dir $GLUE_DIR/$TASK_NAME \
    --max_seq_length 128 \
    --per_device_eval_batch_size=32   \
    --per_device_train_batch_size=32   \
    --learning_rate 2e-5 \
    --num_train_epochs 3.0 \
    --overwrite_output_dir  \
    --output_dir ./tmp/filter_nouns_pairs \`

It stands to reason that I should get a prediction file containing 40k rows, but in fact the output result is still only 10,000 rows. Is the result truncated during prediction? Thanks!

**A link to original question on the forum/Stack Overflow**:

**Comment by Deep1994** [comment]
I found the cause of the problem. If you want to predict on your own test set, you need to delete the previous "cached_test_BertTokenizer_128_qqp.lock" file and "cached_test_BertTokenizer_128_qqp" file.
